docs/design/storage-node-deployment-windows-home.md

```diff
@@ -0,0 +1,165 @@
+# Storage Node Automatic Updates and Installation for Windows
+
+## Overview
+
+As more Storage Node Operators join the network we must ensure these nodes have a mechanism to automatically keep their nodes software up to date.
+If a Storage Node Operator fails to keep their node up to date with the minimum version required by satellites they will no longer be selected for upload or download requests.
+
+Currently we are using Docker for automatic updates which means we require Docker to be installed.
+Docker is being used for many other things as well, so we need to cover these cases as well.
+
+## Goals
+
+To deploy the automatic updates we need to handle the following cases that docker handles. Docker is not supported on windows home so we will ensure an automatic update system is built into the Storage nodes. 
+We also need to ensure that it plays nicely with all the common anti-viruses and firewalls.
+
+Docker is being used for:
+
+* Automatic Updates (with watchtower)
+* Restarting on crash
+* Logging (kind of)
+
+## Services
+
+* Installer
+    * Must be run with admin privileges.
+    * Installs the automatic updater binary and error gui application. (msi)
+    * Sets up automatic updater as a windows Service with sufficient privileges.
+* Automatic Updater (binary)
+    * Downloads storage node binary, Sets up storage node, and runs the watchdog process.
+    * Doesn't start the storage node if storage node has not created a valid config.
```

Comment:
How would the user experience be here? You run the installer and nothing works.

We could rather make this part of the installer.